WalletWasabi.Documentation/Guides/DeterministicBuildGuide.md

```diff
@@ -1,67 +1,76 @@
+The previous commands produce Wasabi's binaries for Windows, macOS and Linux. Also, for your convenience, a new file explorer window will navigate you to the binaries location - i.e. `WalletWasabi\\WalletWasabi.Gui\\bin\\dist`.
 
 ![](https://i.imgur.com/8XAQzz4.png)
 
-# 3. Verify Builds
+## 3. Verify builds
 
-You can compare our binaries with the downloads we have on the website: https://wasabiwallet.io/
-In order to end-to-end verify all the downloaded packages you need a Windows, a Linux, and an OSX machine.
+Now, we will attempt to verify the binaries you have just compiled with the officially distributed binaries on https://wasabiwallet.io website. Please download those packages from the website, you should see the following files in your File Explorer:
 
 ![](https://i.imgur.com/aI9Kx0c.png)
 
-## Windows
+### Windows
 
-After you installed Wasabi from the `.msi`, it will be in `C:\Program Files\WasabiWallet` folder. You can compare it with your build:
-
-```sh
-git diff --no-index win7-x64 "C:\Program Files\WasabiWallet"
-```
+* Install Wasabi using `Wasabi-<version>.msi` file. It will install to `C:\Program Files\WasabiWallet` directory.
+* Start `cmd` or Powershell and navigate to the `dist` directory.
+* Execute the following command:
+  ```sh
+  git diff --no-index "win7-x64" "C:\Program Files\WasabiWallet"
+  ```
+  and make sure that there is **NO** difference reported by the command.
 
-## Linux && OSX
+### Linux & macOS
 
-You can use the Windows Subsystem for Linux to verify all the packages in one go. At the time of writing this guide we provide a `.tar.gz` and a `.deb` package for Linux and .dmg for OSX.  
+You can use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/) to verify all the packages in one go. At the time of writing this guide we provide `.tar.gz` and `.deb` packages for Linux and `.dmg` package for macOS.  
 Install the `.deb` package and extract the `tar.gz` and `.dmg` packages, then compare them with your build.
 
-After installing WSL, just type `wsl` in explorer where your downloaded and built packages are located.
+After [installing WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10), just type `wsl` in File Explorer where your downloaded and built packages are located.
 
 ![](https://i.imgur.com/yRUjxvG.png)
 
-### .deb
+#### .deb
 
 ```sh
 sudo dpkg -i Wasabi-1.1.6.deb
 git diff --no-index linux-x64/ /usr/local/bin/wasabiwallet/
 ```
 
-### .tar.gz
+#### .tar.gz
 
 ```sh
 tar -pxzf WasabiLinux-1.1.6.tar.gz
 git diff --no-index linux-x64/ WasabiLinux-1.1.6
 ```
 
-### .dmg
+#### .dmg
 
-You will need to install `7z` (or something else) to extract the `.dmg`: `sudo apt install p7zip-full`
+You will need to install `7z` (or something else) to extract the `.dmg`. You can do that using `sudo apt install p7zip-full` command.
 
 ```sh
 7z x Wasabi-1.1.6.dmg -oWasabiOsx
```

Comment:
Is this not? 

```suggestion
7z x Wasabi-1.1.6.dmg -o WasabiOsx
```

Reply:
* https://linux.die.net/man/1/7z specifies `-o{Directory}`
* https://thedeveloperblog.com/7-zip-examples also uses the syntax `-oDIR`

So no, I don't think it's wise to change that. It may still work but it's better to follow what examples do.